docs/blog/newsletter-august-2020.rst

```diff
@@ -0,0 +1,85 @@
+.. post:: August 02, 2020
+   :tags: newsletter
+
+#######################################
+Write the Docs Newsletter – August 2020
+#######################################
+
+Dear documentarians of the world - good morning, afternoon, or evening! It’s Beth here, writing to you on an absolutely sweltering day in Switzerland. Not going to lie, 36 degrees C in the home office is rather a challenge; on the plus side, at home a mid-afternoon siesta becomes a very viable option… 
+
+The `virtual Portland conference </conf/portland/2020>`__ is fast approaching, on 9-11 August. Tickets are `on the verge of selling out </conf/portland/2020/news/tickets-selling-out/>`__, so if you’ve been sitting on the fence about whether to go, get a move on! `Here’s the schedule </conf/portland/2020/news/schedule/>`__ if it’ll help you make up your mind.
```

Comment:
I think we're calling it online not virtual. Not that it makes a huge difference.

Reply:
my bad, fixed.